### PR TITLE
Card: remove commons type

### PR DESCRIPTION
**Pull request by smhigley**
Parent issue: #22862

Removes Commons type for Card, also removes the unused `focusMode` prop from CardState (looks like it was previously included only because it was in Commons)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4ea56b4243a5348e84fa270b430577886be44527'>4ea56b4243a5348e84fa270b430577886be44527</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/Avatar-mount_base.html">926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/Avatar-mount.html">883</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/Button-mount_base.html">566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/Button-mount.html">551</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProvider-mount_base.html">1863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProvider-mount.html">1855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-mount_base.html">272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-mount.html">278</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">246</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">298</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/MakeStyles-mount_base.html">1567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22872/perf-test-react-components/MakeStyles-mount.html">1570</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-card/src/components/Card/useCard.ts [hunk 1]
```diff
@@ -28,7 +28,6 @@ export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>):
 
   return {
     appearance,
-    focusMode,
```
Comment (Hotell) [inline packages/react-components/react-card/src/components/Card/useCard.ts:31]: not sure if this is ok cc @andrefcdias ?

Comment (andrefcdias) [reply packages/react-components/react-card/src/components/Card/useCard.ts:31]: I'd say it's fine as `focusMode` will only be used inside the hook.

**Review by andrefcdias — APPROVED** [review]

**Pull request merged**